python/tvm/utils/roofline/cuda.py

```diff
@@ -234,3 +259,63 @@ def estimate_peak_bandwidth(
     b = nd.empty((blocks, 4, warp_size), dtype="float32", device=dev)
     times = f.time_evaluator(f.entry_name, dev, repeat=10, number=1)(a, b)
     return a.numpy().size * 4 / times.min  # 4 bytes per float32
+
+
+@registry.estimate_peak_bandwidth.register("cuda")
+def estimate_peak_bandwidth(
```

Comment:
Naming with the new scheme is a little confusing, hard to delineate `estimate_peak_bandwidth_global` and `estimate_peak_bandwidth` and with the other new functions, can you make a better name?

Reply:
Renamed `estimate_peak_bandwidth_global` to `estimate_peak_bandwidth_global_mem`. Hopefully this makes it clear that `estimate_peak_bandwidth_global_mem` estimates the peak bandwidth to global memory. `estimate_peak_bandwidth` estimates the peak bandwidth depending on the primfunc and features.